lib/rules/require-padding-newlines-after-blocks.js

```diff
@@ -72,10 +113,30 @@ module.exports = function() {};
 module.exports.prototype = {
 
     configure: function(value) {
-        assert(
-            value === true,
-            this.getOptionName() + ' option requires a true value or should be removed'
-        );
+        this._allowInCallExpressions = false;
+        this._allowInArrayExpressions = false;
+
+        if (typeof value === 'object') {
```

Comment:
This whole block is very complicated, i guess it would be better to work on https://github.com/jscs-dev/node-jscs/issues/895 instead?